synapsis/src/main/java/com/soen/synapsis/appuser/AppUserService.java

```diff
@@ -80,4 +82,16 @@ public String updatePassword(String email, String password){
             throw new IllegalStateException("This email does not belong to any user.");
         }
     }
+    public void unmarkRecruiterToCandidate(AppUser appUser, @AuthenticationPrincipal AppUser companyUser) {
```

Comment:
Should we check here that the appUser is actually a recruiter for the company, and it's not a company unmarking another company's recruiter?

Reply:
Yes, that would make sense.